app/controllers/users_controller.rb

```diff
@@ -16,6 +16,12 @@ def create
     end
   end
 
+  def show
+    @user = User.find(params[:id])
+    @queue_items = @user.queue_items
```

Comment:
You should have current_user here - so you could go `current_user.queue_items` or even `User.find(params[:id]).queue_items`